x/mint/spec/04_params.md

```diff
@@ -4,41 +4,33 @@ order: 4
 
 # Parameters
 
-The mint module contains the following parameters:
+The `mint` module contains the following parameters:
 
 | Key                  | Type                | Example |
 |----------------------|---------------------|---------|
 | mint_denom           | string              | "stake" |
 | block_time_threshold | time.duration       | "10s"   |
 | inflation_schedules  | []InflationSchedule |         |
 
-
 ## MintDenom
 
 MintDenom is the denomination of the coin to be minted.
 
 ## BlockTimeThreshold
 
-If the difference in block time from the `LastBlockTime` is greater than `BlockTimeThreshold`, the block only generates inflation as long as `BlockTimeThreshold.
-so actual minted amount could be less than the defined `InflationSchedule.Amount` depending on the number of times blocks having the block time length over `BlockTimeThreshold` occurs.
-
-This is to prevent inflationary manipulation attacks caused by stopping chains or block time manipulation.
+It is a parameter to prevent from inflationary manipulation attacks. Although it is highly unlikely to happen, block time manipulation can be possible with a group of malicious validators who have enough voting power. In case block time delays, BlockTimeThreshold will be used for current block time to calculate inflation amount for the block. If the `LastBlockTime` is greater than `BlockTimeThreshold`, the block generates with `BlockTimeThreshold` value. Therefore, it is possible that the actual minted amount is less than the defined inflation schedule amount.
```

Comment:
```suggestion
It is a parameter to prevent from inflationary manipulation attacks. Although it is highly unlikely to happen, block time manipulation can be possible with a group of malicious validators who have enough voting power. In case block time delays, `BlockTimeThreshold` will be used for current block time to calculate inflation amount for the block. If the `LastBlockTime` is greater than `BlockTimeThreshold`, the block generates with `BlockTimeThreshold` value. Therefore, it is possible that the actual minted amount is less than the defined inflation schedule amount.
```